src/tlitest/tlitest-run

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+cd $(dirname $0)
+echo "RUN_PATH=$PWD"
+
+TEST_DIRS="
+../../lib
+/usr/local/lib
+/lib
+"
+
+for TEST_DIR in $TEST_DIRS; do
+    TEST_FILE=$(find $TEST_DIR -name test_tlog.py|head -1)
```

Comment:
You don't need to loop over the directories yourself, just pass all of them to `find`.
I.e.:

    find $TEST_DIRS -name test_tlog.py

Reply:
That's very useful to know.